sql-statements/sql-statement-show-variables.md

```diff
@@ -10,17 +10,10 @@ This statement shows a list of variables for the scope of either `GLOBAL` or `SE
 
 ## Synopsis
 
-**ShowStmt:**
-
-![ShowStmt](/media/sqlgram/ShowStmt.png)
-
-**ShowTargetFilterable:**
-
-![ShowTargetFilterable](/media/sqlgram/ShowTargetFilterable.png)
-
-**GlobalScope:**
-
-![GlobalScope](/media/sqlgram/GlobalScope.png)
+```ebnf+diagram
+ShowVariablesStmt ::=
+    "SHOW" ("GLOBAL" | "SESSION")? VARIABLES ( "LIKE" stringLit | "WHERE" ("Variable_name" | "Value") "=" stringLit)?
```

Comment:
```suggestion
    "SHOW" ("GLOBAL" | "SESSION")? VARIABLES ShowLikeOrWhere
    
ShowLikeOrWhere ::=
    "LIKE" SimpleExpr
|   "WHERE" Expression
```

the `WHERE` part supports more than just `Variable_name = 'xxx'` or `Value = 'xxx'` expressions.